src/main/java/org/folio/search/model/metadata/PlainFieldDescription.java

```diff
@@ -66,19 +70,47 @@
   @JsonProperty("default")
   private Object defaultValue;
 
+  /**
+   * Specifies if plain keyword value should be indexed with field or not. Works only for fulltext fields.
+   */
+  private boolean indexPlainValue = true;
+
+  /**
+   * Provides sort description for field. If not specified - standard rules will be applied for sort field.
+   */
   @JsonProperty("sort")
   private SortDescription sortDescription;
 
   /**
    * Checks if resource description field is multi-language.
    *
-   * @return true if field is must be indexed, false - otherwise
+   * @return true if field is multi-language, false - otherwise
    */
   @JsonIgnore
   public boolean isMultilang() {
     return MULTILANG_FIELD_TYPE.equals(index);
   }
 
+  /**
+   * Checks if resource description field is standard full-text.
+   *
+   * @return true if field is standard full text, false - otherwise
+   */
+  @JsonIgnore
+  public boolean isStandardFulltext() {
+    return STANDARD_FIELD_TYPE.equals(index);
+  }
+
+  /**
+   * Checks if resource description field is full-text.
+   *
+   * @return true if field is standard full text, false - otherwise
+   */
+  @JsonProperty
+  public boolean isFulltext() {
```

Comment:
unclear naming, because field isFulltext is likely to be read as getter of "fulltext" private field. Or can be read similar to others in this class.